app/src/main/java/space/taran/arkmemo/utils/SVGUtils.kt

```diff
@@ -0,0 +1,155 @@
+package space.taran.arkmemo.utils
+
+import android.graphics.Path
+import android.util.Log
+import space.taran.arkmemo.data.viewmodels.DrawPath
+import space.taran.arkmemo.utils.SVG.Tags.viewBox
+import java.util.Stack
+
+class SVG {
```

Comment:
Looks fine as a proof-of-concept, but we should adopt some XML library to improve the code.